### PR TITLE
server API docs 

### PR DESCRIPTION
**Pull request by Nixon-Samuel**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -35,6 +35,56 @@ const nextConfig = {
         permanent: true,
         basePath: false,
       },
+      {
+        source: "/server-side/v2/foundation/basics",
+        destination: "/server-side/v2/introduction/basics",
+        permanent: true
```
Comment (saikatmitra91) [inline next.config.js:41]: set it true post going live and some dogfooding in production?

**Review by saikatmitra91 — CHANGES_REQUESTED** [review]

File: .github/styles/Vocab/HMSVocab/accept.txt [hunk 2]
```diff
@@ -51,4 +50,30 @@ uuid
...
+livestream
+recording_info
+bd0c76fd-1ab1-4d7d-ab8d-bbfa74b620c4
+5ff5881b80b66969e1fb35f6
+app_id
+setVolume
+rtmpURLs
+trackBy
+framerate
+recording!
```
[the remark this answers is not in the document]
Comment (Nixon-Samuel) [reply .github/styles/Vocab/HMSVocab/accept.txt]: removed it

File: .github/styles/Vocab/HMSVocab/accept.txt [hunk 3]
```diff
@@ -51,4 +50,30 @@ uuid
...
+left_at
+TL;DR
+Wowza
+bitrate
+custom_role_url
+startedAt
+meetingURL
+livestream
+recording_info
+bd0c76fd-1ab1-4d7d-ab8d-bbfa74b620c4
```
[the remark this answers is not in the document]
Comment (Nixon-Samuel) [reply .github/styles/Vocab/HMSVocab/accept.txt]: removed it

File: next.config.js (hunk 1 above)
Comment (Nixon-Samuel) [reply next.config.js]: makes sense, I've set it as false for now.

**Review by saikatmitra91 — APPROVED** [review]

**Pull request merged**